<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>737</width>
    <height>459</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QComboBox" name="comboBox_2">
      <item>
       <property name="text">
        <string>add empty row</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>add row with filled data</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="0" column="1">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="0" colspan="7">
     <widget class="QTableView" name="tableView"/>
    </item>
    <item row="0" column="4">
     <widget class="QPushButton" name="pushButton_3">
      <property name="text">
       <string>delete</string>
      </property>
     </widget>
    </item>
    <item row="0" column="6">
     <widget class="QPushButton" name="pushButton_4">
      <property name="text">
       <string>extra info</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="pushButton_2">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>add</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QComboBox" name="comboBox">
      <item>
       <property name="text">
        <string>World rank</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Name</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Country</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>National rank</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Quality of education</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Alumni employment</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Quality of faculty</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Publications</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Influence</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Citiations</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Patents</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Score</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Year</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="0" column="3">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="5">
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="1" colspan="2">
     <widget class="QLineEdit" name="lineEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>737</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout_2"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionClose"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAbout_2">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actiondasd">
   <property name="text">
    <string>dasd</string>
   </property>
  </action>
  <action name="actionadas">
   <property name="text">
    <string>adas</string>
   </property>
  </action>
  <action name="actionaaaa">
   <property name="text">
    <string>aaaa</string>
   </property>
  </action>
  <action name="actionaaaa_2">
   <property name="text">
    <string>aaaafff</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
